lighthouse-core/test/audits/time-to-interactive-test.js

```diff
@@ -35,7 +35,7 @@ describe('Performance: time-to-interactive audit', () => {
       assert.equal(output.rawValue, 1105.8, output.debugString);
       assert.equal(output.displayValue, '1105.8ms');
       assert.equal(output.extendedInfo.value.expectedLatencyAtTTI, 20.724);
-      assert.equal(output.extendedInfo.value.timings.fMP, 1099.5);
+      assert.equal(output.extendedInfo.value.timings.fMP, 1099.523);
```

Comment:
Test update because this was truncated to one digit after the decimal point by the `first-meaningful-paint` audit before, now matches the `fmpTiming.toFixed(3)` found in this audit